sdk/common/src/main/java/io/opentelemetry/sdk/common/export/ConfigBuilder.java

```diff
@@ -65,7 +63,9 @@ protected abstract T fromConfigMap(
 
   /** Sets the configuration values from the given {@link Properties} object. */
   public T readProperties(Properties properties) {
-    return fromConfigMap(Maps.fromProperties(properties), NamingConvention.DOT);
+    Map<String, String> map = new HashMap<>(properties.size());
+    properties.forEach((key, value) -> map.put((String) key, (String) value));
```

Comment:
I think a normal foreach loop would be more readable.

Reply:
Alternatively, how about some casting? https://stackoverflow.com/a/17209434/2128694

```java
Map<String, String> map = (Map<String, String>)(Map)properties;
```

Reply:
Thanks!